persistence/sql/migratest/fixtures/login_flow/349c945a-60f8-436a-a301-7a42c92604f9.json

```diff
@@ -1,6 +1,5 @@
 {
   "id": "349c945a-60f8-436a-a301-7a42c92604f9",
-  "oauth2_login_challenge": "3caddfd5-9903-4bce-83ff-cae36f42dff7",
```

Comment:
is this expected?

Reply:
In the migration we delete the old column and add a `oauth2_login_challenge_data` column. See this comment: https://github.com/ory/kratos/pull/3317#discussion_r1229382921

So yes, it is expected, although the open question remains whether we should migrate the data from the old column to the new.

@aeneasr WDYT?